frontend/appflowy_flutter/lib/plugins/database/grid/presentation/widgets/footer/grid_footer.dart

```diff
@@ -15,23 +16,30 @@ class GridAddRowButton extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    final color = Theme.of(context).brightness == Brightness.light
+        ? const Color(0xFF171717).withOpacity(0.4)
+        : const Color(0xFFFFFFFF).withOpacity(0.4);
```

Comment:
In the codebase, there are 4+ similar definitions. All of them are using the same color but with different opacities.

My original idea was to refactor them with `context.xxxColor.withOpacity`, however, in some cases, the opacities are even different in light mode and dark mode.

Reply:
I agree with you, this doesn't look good at all. I can't wait for clearer color definitions